packages/help-center/src/components/utils.tsx

```diff
@@ -35,9 +35,15 @@ export const getLastMessage = ( {
 }: {
 	conversation: OdieConversation | ZendeskConversation;
 } ): OdieMessage | ZendeskMessage | null => {
-	return Array.isArray( conversation.messages ) && conversation.messages.length > 0
-		? conversation.messages[ conversation.messages.length - 1 ]
-		: null;
+	if ( ! Array.isArray( conversation?.messages ) ) {
+		return null;
+	}
+
+	const filteredMessages = conversation.messages.filter(
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		( message ) => ( message as any ).type !== 'form'
+	);
```

Comment:
This way we can avoid bypassing typescript 

```suggestion
	const filteredMessages = conversation.messages.filter( ( message ) =>
		'type' in message ? message.type !== 'form' : true
	);
```